integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForUpdateCommand.scala

```diff
@@ -135,7 +142,7 @@ private[sql] case class CarbonProjectForUpdateCommand(
           CarbonUpdateUtil.cleanUpDeltaFiles(carbonTable, false)
 
           // do delete operation.
-          val segmentsToBeDeleted = DeleteExecution.deleteDeltaExecution(
+          val (segmentsToBeDeleted, updatedRowCountTemp) = DeleteExecution.deleteDeltaExecution(
```

Comment:
why not assign to updatedRowCount directly?

Reply:
It can't assign multiple returned values to a variable, and 'deletedRowCount ' will be returned outside this if branch.